modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/event/EventCutter.scala

```diff
@@ -0,0 +1,5 @@
+package org.tessellation.node.shared.domain.event
+
+trait EventCutter[F[_], A, B] {
+  def cut(scEvents: List[A], dagEvents: List[B]): F[(List[A], List[B])]
```

Comment:
if these are arbitrary types then maby `aEvents` and `bEvents`?

Reply:
done